modules/primer-popover/lib/popover.scss

```diff
@@ -0,0 +1,74 @@
+.Popover {
+  position: absolute;
+  right: 0;
+  left: 0;
+  z-index: 100;
+}
+
+.Popover-message {
+  position: relative;
+  max-width: 235px;
+  margin-right: auto;
+  margin-left: auto;
+
+  // Carets
+  &::before,
+  &::after {
+    position: absolute;
+    left: 50%;
+    display: inline-block;
+    content: "";
+  }
+
+  &::before {
+    top: -15px;
```

Comment:
The border slightly overlaps into the Box area. I tweaked this in inspector (so let me know if I'm missing something) and changing `::before` to `top: -16px` (or even better use `$spacer-3`) and the `::after`  to `-14px` gets rid of the overlap. Because of the alpha transparency on the border you see slightly darker points where the border overlaps but it's barely visible and acceptable to me. Happy to explore a non alpha transparent border option for things like this though.

_Before_
<img width="301" alt="screenshot 2017-11-03 11 11 25" src="https://user-images.githubusercontent.com/334891/32380981-ed33f94a-c087-11e7-919b-e5b306274bd4.png">

_After_
<img width="258" alt="screenshot 2017-11-03 11 05 41" src="https://user-images.githubusercontent.com/334891/32380973-e7c5e2d4-c087-11e7-904b-e3fde3854101.png">